components/event-sources/adapter/http/adapter.go

```diff
@@ -24,7 +24,8 @@ type envConfig struct {
 	EventSource string `envconfig:"EVENT_SOURCE" required:"true"`
 
 	// PORT as required by knative serving runtime contract
-	Port int `envconfig:"PORT" required:"true" default:"8080"`
+	Port           int  `envconfig:"PORT" required:"true" default:"8080"`
+	TracingEnabled bool `envconfig:"TRACING_ENABLED" default:"false"`
```

Comment:
Proposal: let's keep tracing enabled by default and provide an installation-override to disable it if needed